packages/connection/src/class-rest-connector.php

```diff
@@ -155,7 +155,7 @@ public static function connection_status( $rest_response = true ) {
 		$connection_status = array(
 			'isActive'     => $connection->is_active(),
 			'isStaging'    => $status->is_staging_site(),
-			'isRegistered' => $connection->is_registered(),
+			'isRegistered' => $connection->is_connected(),
```

Comment:
@fgiannar do you think we should add a new `isConnected` key here?

Reply:
Maybe we could handle this on a new PR, possibly replacing `isRegistered` front-end as well?

Reply:
I'm not sure if this API is consumed by other plugins that use the connection